applications/portfoliomanager/tests/test_risk_management.py

```
@@ -0,0 +1,628 @@
+            ],  # +1 day, +2 days
+            "close_price": [100.0, 110.0, 120.0],  # +20% cumulative return
+        }
+    )
+
+    current_timestamp = datetime.fromtimestamp(base_timestamp + 172800, tz=UTC)
+
+    result = add_positions_performance_columns(
+        positions, original_predictions, original_equity_bars, current_timestamp
+    )
+
+    assert result["maintain_position"][0] is True  # 20% > 15% threshold
```

Ruff (0.12.2)

Use of `assert` detected

(S101)